cli/lib/util.ts

```diff
@@ -554,7 +558,8 @@ const util = {
 
   isPossibleLinuxWithIncorrectDisplay,
 
-  getGitHubIssueUrl (number) {
+  getGitHubIssueUrl (number: number): string {
+    // @ts-expect-error
```

Comment:
@AtofStryker Can you add comment here for why this is needed?

Reply:
added in f840a2b6722e154bc2aff8170a2fae5a2e7efc06